### ISSUE TITLE
snap community theme

### ISSUE THREAD

**Issue by mariomie** [C2N14/AutomaThemely#9]
Hi, I am running the snap community theme on ubuntu 18.04. Unfortunately the theme is not available in the drop down list of AutomaThemely. Do you see any chance of adding it?

**Comment by C2N14** [comment]
Yeah I can probably add it.
Would you happen to know where it it stored though? Are you able to set it through GNOME Tweaks?

**Comment by mariomie** [comment]
Yes, Communitheme and Communitheme-dark can be set through gnome tweaks. Storage location should be "/snap/communitheme/current/share/themes/Communitheme" and "/snap/communitheme/current/share/themes/Communitheme-dark"

**Comment by mariomie** [comment]
I saw that you added the snap path. I can confirm that it is working for me. Thanks!

**Comment by ofou** [comment]
not working for me on 1.3 though :disappointed:
